src/ophyd_async/fastcs/eiger/_eiger_controller.py

```diff
@@ -51,6 +52,9 @@ async def prepare(self, trigger_info: TriggerInfo):
                     self._drv.detector.frame_time.set(trigger_info.livetime),
                 ]
             )
+
+        await wait_for_value(self._drv.stale_parameters, False, timeout=DEFAULT_TIMEOUT)
```

Comment:
Should: I believe with the changes in https://github.com/DiamondLightSource/fastcs-eiger/pull/61 this is not required. Lets remove for now and see how we go.